doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -12,6 +12,7 @@ v6.6
 
 - **/api/metadatablocks** is no longer returning duplicated metadata properties and does not omit metadata properties when called.
 - **/api/roles**: :ref:`show-role` now properly returns 403 Forbidden instead of 401 Unauthorized when you pass a working API token that doesn't have the right permission.
+- openaire implementation can now correctly process one or multiple productionPlaces as geolocation.
```

Comment:
```suggestion
```

It's confusing but despite the name "changelog" we only document backward-incompatible changes in this file.

As the top of the file says, we use release notes to communicate changes that are backward compatible.

@ffritze can you please add a release note snippet? Here's how: https://guides.dataverse.org/en/6.5/developers/version-control.html#writing-release-note-snippets . That would be a great place to communicate the bug you fixed. (Thanks, by the way!)